### ISSUE TITLE
MZN 2.4.1: bug with logical operations

### ISSUE THREAD

**Issue by matsc-at-sics-se** [MiniZinc/libminizinc#390]
MiniZinc says `WARNING: model inconsistency detected` on this:
```
var bool: p;
constraint
    not(((false /\ true) \/ p));
solve satisfy;
```
even though the constraint is true for `p = false`.

**Comment by a1880** [comment]
This Warning is not issued in my MiniZinc 2.4.3 installation under Windows.

**Comment by cyderize** [comment]
This is a duplicate of #359 and was fixed in version 2.4.2.
